### PR TITLE
add some fractal examples for utility classes

### PR DESCRIPTION
**Pull request by donjo**
Adds some initial structure for documenting utility classes. The docs folder is a place where some of the overviews for utility things can live. I was hoping to create an additional group (like Components) for utilities but that may not be possible in this version of fractal.

With that said, I think we could do some initial cleanup and grouping on the fractal components that we document to make a shorter list of high level categories to make utilities easier to find. As it stands now they pretty much mirror how things are documented on `uswds-site` but I don't think that's necessary anymore given how we use the fractal site. 

[😎 Preview](https://federalist-proxy.app.cloud.gov/preview/uswds/uswds/fractal-utility-structure)

### CODE REVIEW

File: src/components/utilities/color/colors.config.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      - gold
+      - orange
+      - peach
+      - terra
+      - red
+      - magenta
+      - purple
+      - gray-cool
+      - gray
+      - gray-warm
```
Comment (maya) [inline src/components/utilities/color/colors.config.yml]: @thisisdano mentioned he needs to change the color names, so this would need to change.

**Comment by donjo** [comment]
Hmmm so while this was just intended to be a PR for some placeholder content I'm realizing that it's not going to be able to pass any of the aXe tests unless we manually change the text color some of these color examples. I'd vote for turning off the testing on all the utilities examples since their job is to just be a visualization for the tokens in the system and not actual endorsed implementations. However, I'm not quite sure how to do that (this problem is blocking us on uswds/uswds-site#450 too)

**Comment by maya** [comment]
I'd check with @thisisdano with how he wants colors displayed. If we moved the text off of the color, then the errors won't flag. Want to go through this a bit as we know folks will be looking towards this for accessibility guidance as well.

screen shot 2018-02-06 at 3 03 36 pm
via #2255.

**Comment by donjo** [comment]
Oh yeah, that style makes sense. Just pushed a temporary change to move the text out of the backgrounds so it will past tests but we can talk about better ways to display that (maybe like the sketch example above)

**Pull request closed without merging**